src/bindings/python/tests/test_graph/test_pooling.py

```diff
@@ -32,6 +32,66 @@ def test_avg_pool_2d(ndarray_1x1x4x4):
     assert node.get_output_element_type(0) == Type.f32
 
 
+def test_avg_pooling_3d_ceil_torch():
+    data = np.arange(0, 25, dtype=np.float32).reshape(1, 1, 5, 5)
+    param = ov.parameter(list(data.shape))
+    kernel_shape = [2, 2]
+    strides = [2, 2]
+    pads_begin = [1, 1]
+    pads_end = [1, 1]
+
+    node = ov.avg_pool(param, strides, pads_begin, pads_end, kernel_shape, False, "ceil_torch")
+    assert node.get_type_name() == "AvgPool"
+    assert node.get_output_size() == 1
+    assert list(node.get_output_shape(0)) == [1, 1, 3, 3]
+    assert node.get_output_element_type(0) == Type.f32
+
+
+def test_avg_pool_2d_ceil_torch():
+    input_data = np.arange(0, 243, dtype=np.float32).reshape(1, 3, 9, 9)
+    param = ov.parameter(input_data.shape, name="A", dtype=np.float32)
+    kernel_shape = [2, 2]
+    strides = [2, 2]
+    pads_begin = [1, 1]
+    pads_end = [1, 1]
+
+    node = ov.avg_pool(param, strides, pads_begin, pads_end, kernel_shape, False, "ceil_torch")
+    assert node.get_type_name() == "AvgPool"
+    assert node.get_output_size() == 1
+    assert list(node.get_output_shape(0)) == [1, 3, 5, 5]
+    assert node.get_output_element_type(0) == Type.f32
+
+
+def test_avg_pool_3x3_kernel_ceil_torch_strides():
```

Comment:
some of these tests looks pretty similar with different values.
May be use parametrization to reduce the number of repeated checks?

Reply:
Done, the tests have been parametrized.